source/endpoint-reference/endpoint-folders.rst

```diff
@@ -0,0 +1,191 @@
+Member   Type    Description
+======== ======= ===============================================================
+size     int     Indicates the number of total variables stored under this
+                 folder and its subfolders.
+-------- ------- ---------------------------------------------------------------
+body     object  Contains a `name` attribute, which is the folder's name.
+                 The root folder has an empty string as name.
+-------- ------- ---------------------------------------------------------------
+index    object  Behaves like a standard Shoji catalog containing the children
+                 of the folder keyed by URL. This catalog will contain a mixed
+                 set of folder and variables URLs. Clients should differentiate
+                 each of the objects by looking at the tuple's `type` attribute.
+                 Folders will have type "folder" while variables will contain
+                 their variables type.
+-------- ------- ---------------------------------------------------------------
+graph    list    A flat list containing the list of URLs of variables and
+                 subfolders on the given folder.
+======== ======= ===============================================================
+
+JSON example:
+
+.. language_specific::
+   --JSON
+   .. code:: json
+
+      {
+          "element": "shoji:catalog",
+          "self": "https://app.crunch.io/api/datasets/5ee0a0/variables/",
+          "size": 10000,
+          "body": {
+              "name": "Folder name"
+          },
+          "index": {
+              "https://app.crunch.io/datasets/abcdef/variables/123/": {
+                  "name": "Birth Year",
+                  "derived": false,
+                  "discarded": false,
+                  "alias": "birthyear",
+                  "type": "numeric",
+                  "id": "123",
+                  "notes": "",
+                  "description": "In what year were you born?"
+              },
+              "https://app.crunch.io/datasets/abcdef/folders/qwe/": {
+                  "name": "Subfolder name"
+              },
+              "https://app.crunch.io/datasets/abcdef/variables/456/": {
+                  "subvariables_catalog": "aad4ad/subvariables/",
+                  "name": "An Array",
+                  "derived": true,
+                  "discarded": false,
+                  "alias": "arrayvar",
+                  "subvariables": [
+                      "439dcf/",
+                      "1c99ea/"
+                  ],
+                  "notes": "All variable types can have notes",
+                  "type": "categorical_array",
+                  "id": "456",
+                  "description": ""
+              }
+          },
+          "graph": [
+              "https://app.crunch.io/datasets/abcdef/variables/456/",
+              "https://app.crunch.io/datasets/abcdef/variables/123/",
+              "https://app.crunch.io/datasets/abcdef/folders/qwe/"
```

Comment:
Can these all 3 be relative URL's instead of absolute?

Reply:
Should we make relative URLs be the default? (They are in the design doc.)

Reply:
In the API they aren't relative, but can be made so.

Reply:
Made a ticket to deal with this https://www.pivotaltracker.com/story/show/154267059